### PR TITLE
Reformat controller

### LINKED ISSUE #4: Poor endpoint management on java server
Endpoint `/healthy` should only be used to check server availability. ID generation and aftermath should be done in another one.

### PR DESCRIPTION
**Pull request by anamonium**

### CODE REVIEW

File: server/src/main/java/com/demo/Controller.java [hunk 1]
```diff
@@ -33,6 +33,11 @@ public Controller(DaprClient client, MessageRepository messageRepository) {
     @ResponseStatus(code = HttpStatus.OK)
     public String health() {
         logger.debug("-- healthy --");
+        return "healthy";
+    }
+
+    @GetMapping("/generateId")
```
Comment (strjakub) [inline server/src/main/java/com/demo/Controller.java]: According to REST standard, path should point to the resource, not the action. That's why it is unrecommended to use verbs in paths

**Review by strjakub — CHANGES_REQUESTED** [review]
.

File: server/src/main/java/com/demo/Controller.java (hunk 1 above)
Comment (anamonium) [reply server/src/main/java/com/demo/Controller.java]: okey-dokey

File: server/src/main/java/com/demo/Controller.java [hunk 2]
```diff
@@ -33,6 +33,11 @@ public Controller(DaprClient client, MessageRepository messageRepository) {
     @ResponseStatus(code = HttpStatus.OK)
     public String health() {
         logger.debug("-- healthy --");
+        return "healthy";
+    }
+
+    @GetMapping("/generatedId")
```
Comment (strjakub) [inline server/src/main/java/com/demo/Controller.java:39]: I thought about just `/id` but still, cannot complain now xD

**Review by strjakub — APPROVED** [review]
lgtm

**Pull request merged**